.github/workflows/workflow.yaml

```diff
@@ -46,6 +46,16 @@ jobs:
         run: |
           make lint
 
+      - name: Checkout Source
```

Comment:
You shouldn't need this, you can just add `gosec` as part of linters at https://github.com/open-policy-agent/gatekeeper/blob/master/.golangci.yaml#L8

Reply:
done